.github/workflows/backend_tests.yml

```diff
@@ -138,3 +144,53 @@ jobs:
         with:
           message: "Backend coverage checks failed on the upstream develop branch."
           webhook-url: ${{ secrets.BUILD_FAILURE_ROOM_WEBHOOK_URL }}
+  check_backend_test_times:
+    name: Check backend test times
+    needs: run_tests
+    runs-on: ubuntu-22.04
+    steps:
```

Comment:
Isn't it possible to create some kind of loop for this? Or how do we make sure we do not miss a shard if we decide to add a new one.

Reply:
I did something similar to the coverage reports, I believe it would be hard to do a loop for this since its a github action instead of a custom script. We could do a matrix but that would result in the creation of more jobs and would take up more workers than necessary. Regarding making sure we don't miss a shard, I think this would be a rare occurrence and any PR which implements this would have to update the coverage one and also this one.